bindings/go/src/fdb/transaction.go

```diff
@@ -337,6 +337,12 @@ func (t Transaction) Clear(key KeyConvertible) {
 // ClearRange removes all keys k such that begin <= k < end, and their
 // associated values. ClearRange returns immediately, having modified the
 // snapshot of the database represented by the transaction.
+// Range clears are efficient with FoundationDB -- clearing large amounts of data
+// will be fast. However, this will not immediately free up disk -
+// data for the deleted range is cleaned up in the background.
+// This will not affect the database until commit is called.
+// Actual amount of stored data for the specified range does not count against
+// the write transaction size limit.
```

Comment:
It may be helpful to describe what the size of the clear range actually is, e.g.:

For purposes of computing the transaction size, only the begin and end keys of a clear range are counted.
The size of the data stored in the range does not count against the transaction size limit.